.github/workflows/docker-image.yml

```diff
@@ -17,31 +16,38 @@ permissions:
   packages: write
 
 jobs:
-  docker-image:
-    name: Check Docker Image
-    runs-on: ubuntu-22.04
-    env:
-      RUNNER_IMAGE_NAME: ${{ inputs.runner_image_name }}
+  docker-image-handling:
```

Comment:
same as above

Reply:
updated in f7270e29dddbc6b96c720aaca7302044b7b08e54